src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -300,19 +301,22 @@ export default class AircraftCommander {
      * @return {array} [success of operation, readback]
      */
     runHold(aircraft, data) {
-        const turnDirection = data[0];
-        const legLength = data[1];
-        const fixName = data[2];
+        const [turnDirection, legLength, fixName, radial] = data;
         const fixModel = NavigationLibrary.findFixByName(fixName);
 
         if (!fixModel) {
             return [false, `unable to hold at unknown fix ${fixName}`];
         }
 
+        // Radial is given as the outbound course, so it needs to be inverted
+        const inboundHeading = radial === null ?
+            fixModel.positionModel.bearingFromPosition(aircraft.positionModel) :
+            radians_normalize(degreesToRadians(radial) + Math.PI);
+
         const holdParameters = {
             turnDirection,
             legLength,
-            inboundHeading: fixModel.positionModel.bearingFromPosition(aircraft.positionModel)
+            inboundHeading: inboundHeading
```

Comment:
Can actually just do `inboundHeading,` here, since the key and variable name are the same :wink: